doc/en/components/inputs/file-input.md

```diff
@@ -0,0 +1,165 @@
+* `autofocus` -	Automatically focuses the file input field when the page loads.
+* `label` -	Sets the label text associated with the file input element.
+* `placeholder` -	Provides placeholder text displayed when no file is selected.
+
+```html
+  <igc-file-input 
+    label="Files Input" 
+    accept=".jpg, .png, .gif"
+    placeholder="Files missing"
+    required
+    multiple>
+  </igc-file-input>
+```
+
+### Methods 
+
+In addition to its configurable properties, the File Input component also inherits four useful methods from the `Input` component that you can use: 
+
+* `focus()` - Sets the focus on the file input element.
+* `blur()` - Removes the focus from the file input element.
+* `reportValidity()` - Checks the validity of the input and displays a validation message if the input is invalid.
+* `setCustomValidity()` - Sets a custom validation message. If the provided message is not empty, the input will be marked as invalid.
+
+### Slots
+
+The File Input component also exposes several slots that can be used to customize its appearance and behavior.
+
+* `prefix` & `suffix` - Allow you to insert content before or after the main input area. 
+* `helper-text` - Displays a hint or instructional message below the input. Useful for providing additional guidance, such as formatting tips or field requirements.
+* `file-selector-text` - Allow you to customizes the text displayed on the file selection button.
+* `file-missing-text` - Sets the text shown in the input field when no file has been selected.
+* `value-missing` -  Renders custom content when the required field validation fails. (i.e., when a file is required but not provided).
+* `invalid` – Allows you to render custom content when the input is in an invalid state.
+* `custom-error` - Displays content when a custom validation message is set using the `setCustomValidity()` method.
+
+`sample="/inputs/file-input/preffix-suffix", height="120", alt="{Platform} File Input Slots Example"`
+
+## Integration
+
+The File Input component integrates seamlessly with the HTML Form element. Using the methods and properties described above, you can effectively manage its behavior and validation within the standard HTML Forms.
+
+`sample="/inputs/file-input/form", height="120", alt="{Platform} File Input Form Example"`
+
+## Limitations
+
+The File Input component currently has the following limitations:
+- The default text for the "Browse" button and the "No file chosen" message is not automatically localized. These texts remain the same across all locales but can be manually customized using the appropriate slots or placeholder binding.
+- Files cannot be set manually through the `value` property. File selection can be done only via the file picker.
+- When the `multiple` property is set, only the first selected file will be shown in the element value.
```

Comment:
```suggestion
```

This is not true. We attempt to show all item names. However, if the first item name is too long, the text will be truncated.